### PR TITLE
fixes #418

### LINKED ISSUE #418: "Count Issues" does not show "closed".
After pressing "Count Issues", issue status is always "missing".
http://garnet.wdidc.org/assignments/63?cohort=1

It indicates that Dan's issue is "missing", but it is "closed" (https://github.com/ga-wdi-exercises/pixart_js/issues?q=involves:dan-ator)

### PR DESCRIPTION
**Pull request by jshawl**
should probably replace https://github.com/ga-dc/garnet/blob/ede575b801b5316f0f06a505d5936618741c384e/app/models/assignment.rb#L46 because the output is not human readable

### CODE REVIEW

File: app/assets/javascripts/assignments.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ function loadIssues(url, callback){
 
   function parseIssue(issue){
     var userID, state;
-    userID = issue[8][1][1][1];
-    state = issue[10][1];
+    userID = issue[9][1][1][1];
+    state = issue[11][1];
```
Comment (RobertAKARobin) [inline app/assets/javascripts/assignments.js]: Could you add a comment saying what `issue[9][1][1][1]` and `issue[11][1]` return?

Comment (jshawl) [inline app/assets/javascripts/assignments.js]: user id and state?

Comment (jshawl) [inline app/assets/javascripts/assignments.js]: in the variable names!

Comment (RobertAKARobin) [inline app/assets/javascripts/assignments.js]: Doh.

**Comment by RobertAKARobin** [comment]
:+1:

**Pull request merged**